chapter_convolutional-neural-networks/lenet_vn.md

```diff
@@ -72,7 +75,14 @@ Each convolutional layer uses a $5\times 5$ kernel and processes each output wit
 The first convolutional layer has 6 output channels, and second convolutional layer increases channel depth further to 16.
 -->
 
-*dịch đoạn phía trên*
+Các đơn vị cơ bản trong khối tích chập là một tầng tích chập và một lớp gộp trung bình tiếp theo
+(lưu ý rằng gộp lớn nhất hoạt động tốt hơn, nhưng nó chưa được phát minh vào những năm 90).
+Tầng tích chập được sử dụng để nhận dạng các mẫu không gian trong ảnh,
+chẳng hạn như các đường kẻ và các bộ phận của các đối tượng và lớp gộp trung bình tiếp theo được sử dụng để làm giảm tính chiều.
+Khối tầng tích chập gồm các ngăn xếp lặp lại của hai đơn vị cơ bản này.
```

Comment:
```suggestion
Khối tầng tích chập tạo nên từ việc xếp chồng các khối nhỏ gồm hai đơn vị cơ bản này.
```

Reply:
"The convolutional layer block is composed of repeated stacks of these two basic units."

Có thể giải thích cho mình tại sao bạn gợi ý dịch ra như ở phía trên được ko? Cảm ơn bạn.

Reply:
Ở đây mình dịch theo nghĩa của cả câu tiếng Anh, không dịch word by word. Ý cả câu là cả khối tầng tích chập này gồm có nhiều (Conv2D + AvgPool2D) nối lại với nhau. Bạn có thể kiểm chứng lại nghĩa này ở hình vẽ phía trên `../img/lenet.svg`.

Reply:
Mình đồng ý. Mình resolve convo này nhé. Thanks bạn.